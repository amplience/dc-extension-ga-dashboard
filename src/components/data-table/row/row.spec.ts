import { render } from '@testing-library/svelte';
import Row from './row.svelte';

describe('Row', () => {
  it('should render the DataTable Row component', () => {
    const { container } = render(Row, {});

    expect(container.firstChild).toMatchSnapshot();
  });
});
