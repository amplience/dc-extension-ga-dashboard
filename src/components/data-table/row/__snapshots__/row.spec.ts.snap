// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render the DataTable Row component 1`] = `
<div>
  <div
    class="data-table-row svelte-hige1l"
  />
</div>
`;
